src/components/Table/Table.scss

```diff
@@ -55,6 +55,7 @@
         border-bottom: 1px solid var(--g-color-line-generic);
 
         line-height: variables.$tableLineHeight;
+        vertical-align: top;
```

Comment:
That is a breaking change. We use default alignment for column (which is center). This behaviour can be changed via `verticalAlign` prop

Reply:
[fixed](https://github.com/gravity-ui/uikit/pull/1149/files#diff-e550fc4a251be96df10c117d97a0b9525d3ed80d4177be7d2169fa7518e4fa0cR106)